examples/html-host/index.html

```diff
@@ -37,78 +37,40 @@
 
   <body>
     <script type="module">
-      // Event Bus
-      import { events } from '@dropins/tools/event-bus.js';
+      
+      // Import API functions to use with action buttons
+      // Replace `my-pkg` with `@dropins/YourDropin`
+      import * as pkg from 'my-pkg/api.js';
 
-      // GraphQL Client
-      import * as mesh from '@dropins/tools/fetch-graphql.js';
+      // The event bus is a core communication tool for all dropins.
+      // They subscribe and publish to events to talk to each other without direct coupling.
+      import { events } from '@dropins/tools/event-bus.js';
 
       // Initializer
       import { initializers } from '@dropins/tools/initializer.js';
 
-      // My Domain Package
-      import * as pkg from 'my-pkg/api.js';
+      // GraphQL Client
+      import * as mesh from '@dropins/tools/fetch-graphql.js';
 
       // import { render as provider } from 'my-pkg/render.js';
       // import <Container> from 'my-pkg/containers/<Container>.js';
 
       // Initialize GraphQl Client (Mesh)
       // Mesh Endpoint
-      mesh.setEndpoint(process.env.ENDPOINT);
+      mesh.setEndpoint('mock-endpoint');
```

Comment:
@A-wakil can you refresh me on why we are changing this to 'mock-endpoint' vs process.env.ENDPOINT?

Reply:
Thanks for clarifying in standup and that documentation would be connected to this.